### PR TITLE
Add pullapproverc: either cchan or lpan commenting "lgtm" will approv…

### PR DESCRIPTION
**Pull request by cchan**
…e the pull

Theoretically could change required: 2 if you want, but nah

### CODE REVIEW

**Comment by lpan** [comment]
LGTM :+1: on this one bro! Btw we should make it a bit more fun :wink: maybe "god bless Mr. Goose " or something similar

File: .pullapproverc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+approve_by_comment: true
+approve_regex: '^(lgtm|:shipit:|:\+1:)'
```
Comment (lpan) [inline .pullapproverc]: :shipit:

**Comment by lpan** [comment]
Dank Mr. Goose

**Comment by cchan** [comment]
Dank Mr Goose

**Pull request merged**